diff --git a/src/komega/CMakeLists.txt b/src/komega/CMakeLists.txt
index bbb42633..a1fda843 100644
--- a/src/komega/CMakeLists.txt
+++ b/src/komega/CMakeLists.txt
@@ -5,9 +5,14 @@ if(${CMAKE_PROJECT_NAME} STREQUAL "Project")
 endif(${CMAKE_PROJECT_NAME} STREQUAL "Project")
 
 set(SOURCES_KOMEGA komega_bicg.F90 komega_math.F90 komega_vals.F90)
-add_definitions(${MPI_Fortran_COMPILE_FLAGS})
-include_directories(${MPI_Fortran_INCLUDE_PATH})
-link_directories(${MPI_Fortran_LIBRARIES})
+if(MPI_FOUND)
+  add_definitions(${MPI_Fortran_COMPILE_FLAGS})
+  include_directories(${MPI_Fortran_INCLUDE_PATH})
+  link_directories(${MPI_Fortran_LIBRARIES})
+endif(MPI_FOUND)
 add_library(komega ${SOURCES_KOMEGA})
-target_link_libraries(komega ${MPI_Fortran_LIBRARIES} ${LAPACK_LIBRARIES})
+target_link_libraries(komega ${LAPACK_LIBRARIES})
+if(MPI_FOUND)
+  target_link_libraries(komega ${MPI_Fortran_LIBRARIES})
+endif(MPI_FOUND)
 install(TARGETS komega ARCHIVE DESTINATION lib LIBRARY DESTINATION lib RUNTIME DESTINATION bin)
